crates/jstz_cli/src/sandbox/daemon.rs

```diff
@@ -201,7 +201,7 @@ impl Sandbox {
                         }
 
                         // yield point
-                        tokio::time::sleep(Duration::from_millis(100)).await;
+                        tokio::time::sleep(Duration::from_millis(300)).await;
```

Comment:
Why the increase?

Reply:
Oops, I was trying to debug why the sandbox was becoming flakey on operation injection, have reverted. 

Turns out that if the commitment period is too low, the rollup spends all its time creating commitment hashes and stops responding to RPC requests.

Reply:
Lmao, we should probably tell the rollups team about that